### ISSUE TITLE
[Bug] Don't dump URLs in the logs for play store builds

### ISSUE THREAD

**Issue by agi** [mozilla-mobile/fenix#14735]
We shouldn't log full URLs for play store builds, e.g.

```
09-03 23:34:25.979 27227 27227 D GeckoSession: handleMessage GeckoView:LocationChange uri=https://www.instagram.com/<redacted>
```

This can be achieved easily by removing `Log.d` and `Log.i` entirely from the build, FxR does this: https://searchfox.org/mozilla-mobile/rev/d7c5ad84a23415853b1da7e36be37bff9694aa71/FirefoxReality/app/proguard-rules.pro#23-31

**Comment by mcarare** [comment]
@agi Should we also remove other verbosity logging levels? (`v`, `w`, `e `?)

**Comment by agi** [comment]
Yes for `v`  (verbose). We should keep `w` (warnings) and `e` (errors) as they're often very helpful when dealing with user reports.

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on both **`Nightly 201014`** and **`82.0.0-beta.5`** using a OnePlus A3 (Android 6.0.1)
